recipes/regiis.rb

```diff
@@ -28,4 +28,5 @@
   command "%WINDIR%\\Microsoft.Net\\Framework64\\v4.0.30319\\aspnet_regiis.exe -i -enable > #{guard_file}"
   creates guard_file
   only_if 'sc.exe query W3SVC'
+  not_if { File.exist?(guard_file) }
```

Comment:
I'm sorry, you are right, I was mistaking the output from the only_if that shows up in the chef-client run as the command running but just realized it's the the query from the only_if part.

![screen shot 2017-05-04 at 11 12 11 am](https://cloud.githubusercontent.com/assets/3476391/25713554/b6a77cb0-30ba-11e7-899c-f272c66cd5e2.png)